src/components/ExecutorDetails.js

```diff
@@ -0,0 +1,311 @@
+            setPageSize(options.per_page);
+            inventory.current.onRefreshData(options);
+        }
+    };
+
+    useEffect(() => {
+        loadInventory();
+        loadRemediation(id);
+        getPlaybookRun(id, run_id);
+
+    }, []);
+    useEffect(() => {
+        if (playbookRun && playbookRun.data) {
+            setExecutor(playbookRun.data.executors.find(executor => executor.executor_id === executor_id) || {});
+        }
+
+        getPlaybookRunSystems(id, run_id, executor_id);
+    }, [ playbookRun ]);
+
+    useEffect(() => {
+        setSystems(() => playbookRunSystems.map(({ system_id, system_name, status }) => ({
+            id: system_id,
+            display_name: system_name,
+            status,
+            children: <Skeleton size='lg' />
+        })));
+    }, [ playbookRunSystems ]);
+
+    useEffect(() => {
+        setSystems(() => ([
+            ...systems.filter(system => system.id !== playbookRunSystemDetails.system_id),
+            { ...systems.find(system => system.id === playbookRunSystemDetails.system_id),
+                isOpen: true,
+                children: <SyntaxHighlighter language="yaml" style={ dark }>
+                    { playbookRunSystemDetails && playbookRunSystemDetails.console || '' }
+                </SyntaxHighlighter>
+            }
+        ]));
+    }, [ playbookRunSystemDetails ]);
+
+    const systemsStatus = playbookRunSystems.reduce((acc, { status }) => ({ ...acc, [normalizeStatus(status)]: acc[normalizeStatus(status)] + 1 })
+        , { pending: 0,
+            running: 0,
+            success: 0,
+            failure: 0,
+            canceled: 0 });
+
+    const renderMain = (status) => ({
+        running: <Main>
+            <Stack gutter="md">
+                <Card>
+                    <CardHeader className='ins-m-card__header-bold'>
+                        <Button
+                            variant='link' onClick={ () => downloadPlaybook(remediation.id) }>
+                            Download Playbook
+                        </Button>
+                    </CardHeader>
+
+                    <CardBody>
+                        { InventoryTable && <InventoryTable
+                            ref={ inventory }
+                            items={ orderBy(systems, [ s => getSystemName(s), s => s.id ]) }
+                            onRefresh={ onRefresh }
+                            page={ page }
+                            total={ systems.length }
+                            perPage={ pageSize }
+                            tableProps={ { onSelect: undefined } }
+                            expandable
+                            onExpandClick={ (_e, _i, isOpen, { id }) => {
+                                if (isOpen) {
+                                    getPlaybookRunSystemDetails(remediation.id, run_id, id);
+                                    setRefreshInterval(setInterval(() => getPlaybookRunSystemDetails(remediation.id, run_id, id), 10000));
```

Comment:
When there is an update the table switches to skeleton and then renders again. Would it be possible just for the log to get updated without the "reload" effect?